builtin/providers/rundeck/provider.go

```diff
@@ -0,0 +1,50 @@
+package rundeck
+
+import (
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/terraform"
+
+	"github.com/apparentlymart/go-rundeck-api/rundeck"
+)
+
+func Provider() terraform.ResourceProvider {
+	return &schema.Provider{
+		Schema: map[string]*schema.Schema{
+			"url": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "URL of the root of the target Rundeck server.",
+			},
```

Comment:
Any reason not to provide a `RUNDECK_URL` env var here? I'm personally all about fully env-var-configurable providers - and I almost never write a `provider {}` block into my configs.